### PR TITLE
response language improved, one response deprecated | /invitelink

### PR DESCRIPTION
**Pull request by engagepy**
The response language is now crisp. 

Invite link seems redundant, hence ineffective to retain. 

/website was absent in help, now included.

### CODE REVIEW

File: pydelhi.py [hunk 1]
```diff
@@ -172,10 +175,10 @@ def github(bot, update):
     chatAction(bot, update, 'http://github.com/pydelhi')
 
 
-def invitelink(bot, update):
-    chatAction(bot, update, '''To prevent spamming we have removed invite
-    link from the group, please ping any one of the admin/moderators of
-    PyDelhi to help you add your friend to the group.''')
+# def invitelink(bot, update):
```
Comment (gaurav-archive) [inline pydelhi.py]: I think we need this but with some modification in the text like `Please ping any one of the admin/moderators of PyDelhi to help you add your friend to the group.`

File: pydelhi.py [hunk 2]
```diff
@@ -172,10 +175,10 @@ def github(bot, update):
...
+#     chatAction(bot, update, '''To prevent spamming we have removed invite
+#     link from the group, please ping any one of the admin/moderators of
+#     PyDelhi to help you add your friend to the group.''')
 
 
 def gethelp(bot, update):
```
Comment (gaurav-archive) [inline pydelhi.py]: We have to add Twitter also.

Comment (engagepy) [reply pydelhi.py]: I'll do it  !

Comment (gaurav-archive) [reply pydelhi.py]: find it on the master branch.

Comment (gaurav-archive) [reply pydelhi.py]: LinkedIn is missing, I think :)

File: pydelhi.py (hunk 1 above)
Comment (realslimshanky) [reply pydelhi.py]: Agreed.

**Review by realslimshanky — COMMENTED** [review]
Please uncomment `def invitelink ...` Rest LGTM.

**Pull request closed without merging**